### PR TITLE
Merge Martins 0.13.1 work

### PR DESCRIPTION
**Pull request by joshuahuburn**
Added a Visual Logger manual test gym, and associated manual test notes.

### CODE REVIEW

File: USER_MANUAL.md [hunk 1]
```diff
@@ -352,16 +352,21 @@ These test whether key trace events have the appropriate cause events. They can
...
   * Press `K` to trigger a ClientTravel for the PlayerController to the same deployment. If you've moved your camera, pressing `K` should visibly snap the camera back to the position that the camera spawned in (indeed, it is spawning again). If this happens, the test has passed.
 
 ##### Multiworker World Composition gym
-* Tests server's without authoritive player controllers still replicate relevant actors
-* Validation
-  * Enable the replication graph
-    * Before booting the editor, navigate to DefaultEngine.ini and uncomment the `ReplicationDriverClassName` option
-  * Play with 1 client connect.
-  * The client should seem a sublevel actor spawned in each server worker, moving back and forth across the worker boundary. On each cross, the authority of the cube should switch to the appropriate server.
+* Tests that servers without authoritive player controllers are still able to replicate relevant actors.
```
Comment (martin-improbable) [inline USER_MANUAL.md:359]: these are not my changes?

File: USER_MANUAL.md [hunk 2]
```diff
@@ -456,5 +461,62 @@ Manual steps:
   * In the Unreal Editor Toolbar, click Stop when you're done.
   * Shut down the deployment if this doesn't happen automatically.
 * These tests can also be run in the cloud by deploying the `PlayerDisconnectGym` map and launching two clients.
+
+####RPCTimeoutTestGym
```
Comment (martin-improbable) [inline USER_MANUAL.md:465]: I didn't add this, I only added #### Visual Logger gym (see below)?

File: USER_MANUAL.md (hunk 1 above)
Comment (joshuahuburn) [reply USER_MANUAL.md:359]: These are the changes Ollie made

**Review by simonsarginson — APPROVED** [review]
LGTM

**Pull request merged**